src/modules/Game.class.js

```diff
@@ -20,25 +39,114 @@ class Game {
    * If passed, the board will be initialized with the provided
    * initial state.
    */
-  constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+  constructor(initialState = Game.deafultInitialState, isForCheck = false) {
+    this.#initialState = initialState.map((row) => [...row]);
+    this.#applyInitialState();
+    this.#isForCheck = isForCheck;
   }
 
-  moveLeft() {}
-  moveRight() {}
-  moveUp() {}
-  moveDown() {}
+  moveLeft() {
+    let isStateChanged = false;
+
+    if (this.#status !== 'playing' && !this.#isForCheck) {
```

Comment:
Consider checking the game status with a method or getter to encapsulate the status logic, which can improve the maintainability of the code.